params/params_tmpl - Kat.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 4
+    
+# e.g. "anas_krystallinus"    
+species.name <- "Kat's bird "
+    
+# e.g. "red"    
+color <- "#009dff "
```

Comment:
I'm not sure if the space in the string here will cause problems, it might be worth removing it.

```suggestion
color <- "#009dff"
```